ckanext/schema/dc_dataset.yaml

```diff
@@ -0,0 +1,321 @@
+  # TODO create custom preset that serves predefined tag list
+  - field_name: tag_string
+    label: Keywords
+    required: false
+    preset: tag_string_autocomplete
+    display_property: dcat:keyword
+    form_placeholder: eg. electricity, policy, land use
+
+  # Will be used to build a filter (e.g. by year)
+  - field_name: temporal_start
+    label: Start date
+    required: false
+    preset: date
+    display_property: dcat:startDate
+    help_text: Earliest date the dataset covers e.g. 2016-01-01.
+    
+  # Will be used to build a filter (e.g. by year)
+  - field_name: temporal_end
+    label: End date
+    required: false
+    preset: date
+    display_property: dcat:endDate
+    help_text: Latest date the dataset covers e.g. 2019-01. Leave blank if dataset is updated to present day.
+
+  # Will be used to build a filter
+  - field_name: temporal_resolution
+    label: Time resolution
+    required: false
+    preset: select
+    choices:
+    - value: sub_second
+      label: Sub-second
+    - value: second
+      label: Second
+    - value: minute
+      label: Minute
+    - value: hour
+      label: Hour
+    - value: day
+      label: Day
+    - value: week
+      label: Week
+    - value: month
+      label: Month
+    - value: quarter
+      label: Quarter
+    - value: year
+      label: Year
+    display_property: dcat:temporalResolution
+    help_text: For time-series data, please select the smallest resolution represented in your dataset. If your dataset resolution falls between the values given here, please select the closest resolution (e.g. for readings every 30 seconds select minute).
+  
+  # Could be used to automate communication with publisher to update their dataset
+  # Could also feed into freshness or future data quality scores
+  - field_name: update_frequency
+    label: Update frequency
+    required: false
+    preset: select
+    choices:
+    - value: hourly
+      label: Hourly
+    - value: daily
+      label: Daily
+    - value: weekly
+      label: Weekly
+    - value: monthly
+      label: Monthly
+    - value: quarterly
+      label: Quarterly
+    - value: annually
+      label: Annually
+    - value: never
+      label: Never
+    display_property: dcat:accrualPeriodicity
+    help_text: Select the closest value that reflects the frequency with which the dataset is updated.
+
+  # Will be used to build a filter
+  - field_name: iso_region
+    label: ISO region
+    required: false
+    preset: select
+    choices:
+    - value: emea
+      label: EMEA
+    - value: apac
+      label: APAC
+    - value: americas
+      label: AMERICAS
+    - value: world
+      label: WORLD
+    display_property: dct:Location
+    help_text: Select the broad region that your dataset covers.
+
+  # Will be used to build a filter
+  # TODO create scheming_countries_choices helper (see: https://github.com/ckan/ckanext-scheming#choices_helper)
+  - field_name: countries
+    label: Countries
```

Comment:
this is where we can use ISO3166
e.g. https://github.com/lukes/ISO-3166-Countries-with-Regional-Codes

Reply:
👍